Makefile

```diff
@@ -15,10 +16,12 @@ ADMIN_MONGO_PASSWORD?=admin
 
 DUMP_FILENAME=dump.tar.gz
 
-all: build re mongo-reg mongo-insert export-all re import-all
+VERSION:=$(shell git rev-parse --short HEAD)
+
+all: build up mongo-reg mongo-insert export-all export-ch export-vm re import-all
```

Comment:
build re mongo-reg mongo-insert export-all re import-all

Reply:
Fixed in bdf6299c3cc59f1cba6b88ae2f8e96250cf60203